Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html

```diff
@@ -1,67 +1,72 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Lists</title>
-    <style>
-      .disc-list {
-        list-style-type: disc;
-      }
-      .mean {
-        color: peru;
-      }
-      .sys {
-        color: royalblue;
-      }
-    </style>
-  </head>
-  <body>
-    <h1>Ambersoft</h1>
-    <ol>
-      <li class="mean">MI-MEAN</li>
-      <ol style="list-style-type: lower-roman; color: peru">
-        <li>MI-MEAN-6</li>
-        <ul class="disc-list">
-          <li>Shweta</li>
-          <li>Ajay</li>
-          <li>Trupti</li>
-        </ul>
 
-        <li>MI-MEAN-7</li>
-        <ul class="disc-list">
-          <li>Sagar</li>
-          <li>Umesh</li>
-        </ul>
-        <li>MI-MEAN-8</li>
-        <ul class="disc-list">
-          <li>Mihika</li>
-          <li>Pallavi</li>
-          <li>Reva</li>
-        </ul>
-        <li>MI-MEAN-9</li>
-        <ul class="disc-list">
-          <br />
-          <li>Anil</li>
-          <br />
-          <li>Manoj</li>
-          <br />
-        </ul>
-      </ol>
-      <li class="sys">SYSTEM</li>
-      <ol class="sys" style="list-style-type: lower-roman">
-        <li>SYSTEM-1</li>
-        <ul class="disc-list">
-          <li>Anu</li>
-          <li>Dipti</li>
-        </ul>
-        <li>SYSTEM-2</li>
-        <ul class="disc-list">
-          <li>Triveni</li>
-          <li>Kalyani</li>
-        </ul>
-      </ol>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Lists</title>
+  <style>
+    .disc-list {
+      list-style-type: disc;
+    }
+
+    .mean {
+      color: peru;
+    }
+
+    .sys {
+      color: royalblue;
+    }
+  </style>
+</head>
+
+<body>
+  <h1>Ambersoft</h1>
+  <ol>
+    <li class="mean">MI-MEAN</li>
+    <ol style="list-style-type: lower-roman; color: peru">
+      <li>MI-MEAN-6</li>
+      <ul class="disc-list">
+        <li>Shweta</li>
+        <li>Ajay</li>
+        <li>Trupti</li>
+      </ul>
+
+      <li>MI-MEAN-7</li>
+      <ul class="disc-list">
+        <li>Sagar</li>
+        <li>Umesh</li>
+      </ul>
+      <li>MI-MEAN-8</li>
+      <ul class="disc-list">
+        <li>Mihika</li>
+        <li>Pallavi</li>
+        <li>Reva</li>
+      </ul>
+      <li>MI-MEAN-9</li>
+      <ul class="disc-list">
+        <br>
+        <li>Anil</li>
+        <br>
+        <li>Manoj</li>
+        <br>
+      </ul>
+    </ol>
+    <li class="sys">SYSTEM</li>
+    <ol class="sys" style="list-style-type: lower-roman">
+      <li>SYSTEM-1</li>
+      <ul class="disc-list">
+        <li>Anu</li>
+        <li>Dipti</li>
+      </ul>
+      <li>SYSTEM-2</li>
+      <ul class="disc-list">
+        <li>Triveni</li>
+        <li>Kalyani</li>
+      </ul>
     </ol>
-  </body>
-</html>
+  </ol>
```

Comment:
remove this ol tage

Reply:
sir please check my latest commit
I am not seeing this ol in my local file

Reply:
also checked on main branch . there is no `<ol>` on line 69 after `</html>

Reply:
this `</ol>` is required